tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -6,18 +6,22 @@
 
 
 test_cases_old_nonoise = [
-    # This should contain configs where a V2 policy is compatible with a V1 env.
+    # This should contain configs where a V2 policy is running in a V1 env.
     # name, policy, action noise pct, success rate
     ['handle-press-side-v1', SawyerHandlePressSideV2Policy(), .0, .05],
+    ['lever-pull-v1', SawyerLeverPullV2Policy(), .0, .0],
```

Comment:
remove these? is this intentional?

Reply:
These test cases are really just here to document how much better the policies do in V2 envs vs V1 envs. For example, `lever-pull` gets 0% success on V1 but 100% success on V2. Also, the tests are marked as skipped. I can remove them entirely if you want.

Reply:
I am a proponent of you keeping these as notes somewhere, but it doesn't make sense for these to have the option for these to run.